src/DiscordApi/DiscordAPI.hpp

```diff
@@ -11,7 +11,7 @@ static const QString clientSecret = QStringLiteral("og4kFEYYimJCaFn0v9R7mIPg45ci
 
 static const auto fakeBrowserUserAgent = "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:82.0) Gecko/20100101 Firefox/82.0";
 
-static const QString baseURL      = QStringLiteral("https://discord.com/api/v9");
+static const QString baseURL      = QStringLiteral("https://discord.com/api/v");
```

Comment:
Why version is deleted?

Reply:
Leaving the version out allows the client to follow the latest API version. [Here](https://discord.com/developers/docs/reference#api-versioning)